app/actions/local/category.ts

```diff
@@ -39,7 +39,7 @@ export async function storeCategories(serverUrl: string, categories: CategoryWit
         }
 
         if (models.length > 0) {
-            await operator.batchRecords(models);
+            await operator.batchRecords(models, 'storeCategories');
```

Comment:
Sadly Hermes does not allow to use `caller.name` as it throws an error, it does work for JSC but that aint an option, so I guess doing it like this is the only option we have.